kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/strimzi/Strimzi.java

```diff
@@ -35,7 +48,41 @@ public class Strimzi {
      */
     public Strimzi(String deploymentNamespace) throws IOException {
         this.deploymentNamespace = deploymentNamespace;
-        deployment = kubeClient().getClient().load(DeploymentUtils.getDeploymentFileFromURL(Environment.STRIMZI_URL));
+        InputStream strimziYaml = DeploymentUtils.getDeploymentFileFromURL(Environment.STRIMZI_URL);
+        InputStream strimziYamlKRaft = configureKRaftModeForStrimzi(strimziYaml);
+        deployment = kubeClient().getClient().load(strimziYamlKRaft);
+    }
+
+    private InputStream configureKRaftModeForStrimzi(InputStream strimziYaml) throws IOException {
+        YAMLFactory yamlFactory = new YAMLFactory();
+        ObjectMapper mapper = new YAMLMapper();
+
+        YAMLParser yamlParser = yamlFactory.createParser(strimziYaml);
```

Comment:
the code had me scratching my head for a while.
renaming the strimziYaml strimziMultiDocumentYaml would help remind people that we're dealing with a multi document stream.  that'll help readers understand why you use readValues rather than readTree.